src/include/daos_srv/ras.h

```diff
@@ -146,7 +146,7 @@ ds_notify_ras_event(ras_event_t id, char *msg, ras_type_t type, ras_sev_t sev,
  * with a '$' to indicate so. See ds_notify_ras_event for parameter
  * documentation.
  */
-void
+void __attribute__((format(printf, 12, 13)))
```

Comment:
(style)  __printf(string-index, first-to-check) is preferred over __attribute__((format(printf, string-index, first-to-check)))